userspace/libsinsp/container.cpp

```diff
@@ -88,7 +88,7 @@ bool sinsp_container_manager::remove_inactive_containers()
 		});
 
 		auto containers = m_containers.lock();
-		if (m_inspector->m_sinsp_stats_v2)
+		if (m_inspector != nullptr && m_inspector->m_sinsp_stats_v2)
```

Comment:
We need to check for `m_inspector != nullptr` before dereferencing the stats_v2 buffer. I was able to reproduce segfault in some common legit code paths. cc @incertum